source/slang/slang-ir-util.cpp

```diff
@@ -645,7 +645,7 @@ void setInsertAfterOrdinaryInst(IRBuilder* builder, IRInst* inst)
     }
 }
 
-bool areCallArgumentsSideEffectFree(IRCall* call)
+bool areCallArgumentsSideEffectFree(IRCall* call, bool useDominanceTree)
```

Comment:
Might want to use an enum to indicate the useDominanceTree ness, such that it's clearer at call sites what the meaning is.

Reply:
Fixed.